### PR TITLE
Update run-taco.md

### PR DESCRIPTION
**Pull request by dmoynihan**
**Things To Remember:** 

1. Be sure to merge into the right branch. The choices are:
* **`master`** if your PR runs on the current version of Tableau.
* **`dev-2020.1`**
* **`2020.2`**
2. You should update at least one of the following:
* The changelog of the project you modified.
* The root `README.md` if the PR is a release of the SDK, Connector Packager, or TDVT.  
=====

### CODE REVIEW

**Comment by dmoynihan** [comment]
Peter Tempest from Prep asked that we add a blurb about running tacos in Prep.

File: docs/run-taco.md [hunk 1]
```diff
@@ -2,24 +2,30 @@
...
 
 ## Run a packaged connector in Tableau Desktop
-1. Copy your packaged connector file (with a .taco filename extension) into your My Tableau Repository/Connectors directory. 
+1. Copy your TACO file into your My Tableau Repository/Connectors directory. 
 1. Launch Tableau Desktop.
 
+## Run a packaged connector in Tableau Prep
+1. Copy your TACO file into your My Tableau Prep Repository/Connectors directory. 
+1. Launch Tableau Prep.
+
```
Comment (pvanderknyff) [inline docs/run-taco.md:16]: What version of Tableau Prep supports tacos? If it's not 2019.4, like for desktop, we should mention it here.

**Review by pvanderknyff — COMMENTED** [review]
Should https://help.tableau.com/current/pro/desktop/en-us/examples_connector_sdk.htm be updated as well? that page is where customers should go to to find out how to use tacos, these pages are targeted more towards connector developers. Not sure the process to update that though.

**Comment by rosswbrown** [comment]
Re-open if still needed

**Pull request closed without merging**